### PR TITLE
Task01 start

### PR DESCRIPTION
**Pull request by RareIteM01**

### CODE REVIEW

File: src/com/example/ppj04/Task01.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package com.example.ppj04;
+
+public class Task01 {
+    public static void main(String[] args) {
+        System.out.println("Task01");
```
Comment (maxline) [inline src/com/example/ppj04/Task01.java:5]: Please implement solution of task 01

Comment (maxline) [inline src/com/example/ppj04/Task01.java:5]: pleas move this logic to the separate method

**Review by maxline — CHANGES_REQUESTED** [review]
Please improve the solution

File: src/com/example/ppj04/Task01.java (hunk 1 above)
Comment (RareIteM01) [reply src/com/example/ppj04/Task01.java:9]: fixed

Comment (RareIteM01) [reply src/com/example/ppj04/Task01.java:9]: fixed

**Pull request merged**